docs/4-software-development-practices/4.5.5-test-automation.md

```diff
@@ -4,36 +4,40 @@ docs/4-software-development-practices/4.5.5-test-automation.md:
   estReadingMinutes: 5
   exercises:
     -
-      name: Create a Jenkins Pipeline to test Go project
-      description: Create a Jenkins server via an existing Dockerfile, add Go as a dependency to the Image. On the Jenkins instance point to your fork of the Bootcamp with Go Unit Tests. Create a pipeline that builds and tests your fork.
+      name: Create a GitHub Workflow to test a Go project
+      description: Create a GitHub Action that will run Unit Tests when a change is pushed
       estMinutes: 180
       technologies:
-      - Jenkins
-      - Docker
+      - GitHub Actions
       - Go
 ---
 
 # Automated Testing
 
-Automated testing is a crucial step to ensure code quality because developed tests are guaranteed to run once new source code is committed. Jenkins is a tool that can help us leverage our tests in a build step. With a testing stage in Jenkins, code quality can be ensured by automatically running the tests that have been written. If tests fail, the build can be marked as unstable and deployment will not occur.
+Automated testing is a crucial step to ensure code quality because developed tests are guaranteed to run once new source code is committed. GitHub Actions is a tool that can help us leverage our tests in a build step. With a testing stage in GitHub Actions, code quality can be ensured by automatically running the tests that have been written. If tests fail, the build can be marked as unstable and deployment will not occur.
 
-## Exercise
+## Exercise 1
 
-1. Locate the Jenkins Dockerfile created in the container section of the bootcamp.
-2. Edit this Dockerfile to add a Go installation to the image.
-3. Create a Freestyle Project that will point to your completed Go Unit Testing exercise from your fork of the Bootcamp.
+1. Create a new action, `test`, like you did for the `build` action in the `GitHub Actions` section of the bootcamp.
```

Comment:
I would remove reference to the `GitHub Actions` section of the bootcamp as I am not sure when that initiative will get finished or added. The bootcamp as it is is a rather gross offender of saying "do x" without introduction to "x". I am ok if we do that in this situation